wordfreq/tokens.py

```diff
@@ -114,15 +122,26 @@ def turkish_tokenize(text, include_punctuation=False):
     return [token.strip("'").casefold() for token in token_expr.findall(text)]
 
 
-mecab_tokenize = None
 def japanese_tokenize(text, include_punctuation=False):
     """
     Tokenize Japanese text, initializing the MeCab tokenizer if necessary.
     """
     global mecab_tokenize
     if mecab_tokenize is None:
-        from wordfreq.japanese import mecab_tokenize
-    tokens = mecab_tokenize(text)
+        from wordfreq.mecab import mecab_tokenize
+    tokens = mecab_tokenize(text, 'ja')
+    token_expr = TOKEN_RE_WITH_PUNCTUATION if include_punctuation else TOKEN_RE
+    return [token.casefold() for token in tokens if token_expr.match(token)]
+
+
+def korean_tokenize(text, include_punctuation=False):
```

Comment:
Perhaps a bit silly for only two methods, but since these two methods are identical except for whether it's "ja" or "ko" that gets passed as a second argument, is it worth pulling everything else out into some sort of `_ja_and_ko_tokenize` method?